lib/karafka/responders/builder.rb

```diff
@@ -3,30 +3,30 @@
 module Karafka
   # Responders namespace encapsulates all the internal responder implementation parts
   module Responders
-    # Responders builder is used to find (based on the controller class name) a responder that
-    # match the controller. This is used when user does not provide a responder inside routing
-    # but he still names responder with the same convention (and namespaces) as controller
+    # Responders builder is used to find (based on the consumer class name) a responder that
```

Comment:
Responders builder is used to finding (based on the consumer class name) a responder that match the consumer. We use it when user does not provide a responder inside routing, but he still names responder with the same convention (and namespaces) as consumer